kinds/dashboard/dashboard_kind.cue

```diff
@@ -121,11 +121,12 @@ lineage: seqs: [
 					target?:   #AnnotationTarget @grafanamaturity(NeedsExpertReview)
 				} @cuetsy(kind="interface")
 
-				// FROM: packages/grafana-data/src/types/templateVars.ts
-				// TODO docs
-				// TODO what about what's in public/app/features/types.ts?
-				// TODO there appear to be a lot of different kinds of [template] vars here? if so need a disjunction
-				#VariableModel: {
+				// Dashboard variables. See https://grafana.com/docs/grafana/latest/variables/variable-types/
+				#VariableModel: #QueryVariableModel | #AdHocVariableModel | #ConstantVariableModel | #DataSourceVariableModel | #IntervalVariableModel | #TextBoxVariableModel | #CustomVariableModel | #UserSystemVariableModel | #OrgSystemVariableModel | #DashSystemVariableModel @cuetsy(kind="type") @grafana(TSVeneer="type")
```

Comment:
I don't understand what OrgSystemVariableModel, DashSystemVariableModel and UserSystemVariableModel are doing here they are never persisted, just runtime "macros"

Reply:
Yes, they should be living in `@grafana/data` only. I didn't realize they are runtime only. Thanks Torkel